agent/checks/check.go

```diff
@@ -666,6 +695,20 @@ type CheckGRPC struct {
 	stop     bool
 	stopCh   chan struct{}
 	stopLock sync.Mutex
+
+	// Set if checks are exposed through Connect proxies
+	// If set, this is the target of check()
+	ProxyGRPC string
+}
+
+func (c *CheckGRPC) CheckType() structs.CheckType {
+	return structs.CheckType{
+		CheckID:   c.CheckID,
+		GRPC:      c.GRPC,
+		ProxyGRPC: c.ProxyGRPC,
+		Interval:  c.Interval,
+		Timeout:   c.Timeout,
```

Comment:
Same comment here re mentioning it along with `structs.CheckType`.

Reply:
done!